tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
 envlist =
-    pycodestyle-{py27,py37},
+    pycodestyle-{py27,py37,py38}
```

Comment:
This probably needs to be updated with recent changes on `main` that dropped a bunch of outdated Django and Python versions.